### PR TITLE
replay and testnet has been added

### PR DESCRIPTION
**Pull request by Persistencesujith**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,13 +4,14 @@ go 1.12
 
 require (
 	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // indirect
-	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
+	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
 	github.com/cosmos/cosmos-sdk v0.37.0 // latest
-	github.com/cosmos/go-bip39 v0.0.0-20180618194314-52158e4697b8
+	github.com/cosmos/gaia v1.0.0
```
Comment (migueldingli1997) [inline go.mod:9]: This dependency is not necessary. Please revert any other unnecessary dependency changes that might have resulted from this.

File: cmd/ixod/main.go [hunk 2]
```diff
@@ -56,6 +56,7 @@ func main() {
 	rootCmd.AddCommand(genaccscli.AddGenesisAccountCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
 	rootCmd.AddCommand(oraclesCli.AddGenesisOracleCmd(ctx, cdc, app.DefaultNodeHome, app.DefaultCLIHome))
 	rootCmd.AddCommand(client.NewCompletionCmd(rootCmd, true))
+	rootCmd.AddCommand(replayCmd())
```
[the remark this answers is not in the document]
Comment (Persistencesujith) [reply cmd/ixod/main.go:59]: added

File: cmd/ixod/testnet.go [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+		},
+	}
+
+	cmd.Flags().Int(flagNumValidators, 4,
+		"Number of validators to initialize the testnet with")
+	cmd.Flags().StringP(flagOutputDir, "o", "./mytestnet",
+		"Directory to store initialization data for the testnet")
+	cmd.Flags().String(flagNodeDirPrefix, "node",
+		"Prefix the directory name for each node with (node results in node0, node1, ...)")
+	cmd.Flags().String(flagNodeDaemonHome, "ixo",
```
[the remark this answers is not in the document]
Comment (Persistencesujith) [reply cmd/ixod/testnet.go:79]: added ixod

File: go.mod (hunk 1 above)
Comment (Persistencesujith) [reply go.mod:9]: removed

**Pull request merged**